technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java

```diff
@@ -0,0 +1,51 @@
+package com.github.mizool.technology.jackson;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.ParamConverterProvider;
+import javax.ws.rs.ext.Provider;
+
+import com.google.common.base.Strings;
+
+@Provider
+public class LocalDateTimeParamConverterProvider implements ParamConverterProvider
+{
+    @Override
+    public <T> ParamConverter<T> getConverter(Class<T> rawType, Type genericType, Annotation[] annotations)
+    {
+        ParamConverter<T> result = null;
+
+        if (rawType.equals(LocalDateTime.class))
+        {
+            result = new ParamConverter<T>()
```

Comment:
This looks nearly identical to the `ParamConverter` inside the other two provider. Can we instead have a package local class that has a `DateTimeFormatter` constructor argument and is shared between the three providers?

Reply:
I don't see how, given the "ZonedDateTime.from" (etc) calls...